src/encoded/static/components/app.js

```diff
@@ -213,6 +213,25 @@ EulaModal.propTypes = {
 };
 
 
+const AccountCreatedModal = ({ closeModal }) => (
+    <Modal>
+        <ModalHeader title="Account Created" closeModal={closeModal} />
+        <ModalBody>
+            <p>
+                The account was created and you were automatically logged-in.
```

Comment:
I won’t request changes for this, but in case you feel like it, I _think_ “logged-in” in this case should be “logged in.” You’d only use the hyphen when used as an adjective phrase, like “logged-in user.”

Reply:
Fixed.